src/socketHandler/gameHandler.js

```diff
@@ -0,0 +1,37 @@
+const { startRound } = require('../services/game.service');
+const { createGame } = require('../services/gamestate.service');
+const { getByLobbyID } = require('../services/lobby.service');
+
+const startGame = async function (socket, io) {
+  const room = Array.from(socket.rooms).pop();
```

Comment:
I think we should use service functions to get the room id. If the user is in more than one room this could fail.

lobbyservice: getCurrentLobby